services/github/src/handlers.js

```diff
@@ -504,17 +490,56 @@ async function statusHandler(message) {
     `Attempting to update status of the checkrun for ${organization}/${repository}@${sha} (${taskState.conclusion})`,
   );
   try {
+    const taskDefinition = await this.queueClient.task(taskId);
+    debug(`Result status. Got task build from DB and task definition for ${taskId} from Queue service`);
+    const {customCheckRun} = (taskDefinition.extra && taskDefinition.extra.github) || {};
+
+    let customCheckRunText = '';
+    if (customCheckRun && customCheckRun.textArtifactName) {
+      const scopedQueueClient = this.queueClient.use({
+        authorizedScopes: taskDefinition.scopes,
+        credentials: this.context.cfg.taskcluster.credentials,
+      });
+
+      const url = this.queueClient
+        .buildUrl(scopedQueueClient.getArtifact, taskId, runId, customCheckRun.textArtifactName);
+
+      let res = await utils.throttleRequest({url, method: 'GET'});
+
+      if (res.status >= 400) {
+        /*
+          Some possible errors:
+            - insufficient scopes: statusCode 403
+            - no artifact entity in DB, no artifact in S3: statusCode 404
+        */
+        await this.createExceptionComment({
+          debug,
+          instGithub,
+          organization,
+          repository,
+          sha,
+          error: res.response.error,
+        });
```

Comment:
Even with the retries we can still get a 500 here. Probably makes sense to only report 400s directly and instead just say "internal error" for 500s? I could be convinced differently.

Reply:
My rationale for this kind of reporting was these are the things the users can fix or, in case of some force-majeure just an explanation why their custom input didn't appear. If you mean the appearance of the error, they all will have the same appearance: `Cannot GET <url to the artifact> (<errorcode>)`

Reply:
The only issue is that the user cannot fix a 500